### ISSUE TITLE
Unable to receive BCH testnet transactions

### ISSUE THREAD

**Issue by jonas-lundqvist** [bitpay/copay#9827]
I installed the latest BitPay version from Google Play Store (5.9.1) and created a new "Basic Wallet" BCH testnet wallet.
I then went to two different faucets (https://coinfaucet.eu/en/bch-testnet/ and https://developer.bitcoin.com/faucets/bch/) and sent two transactions to my newly created wallet:
* https://explorer.bitcoin.com/tbch/tx/6af6fb047b9665511d41e2fa7d171ddce4e2abf8bc2cc09590b2087b59791a93
* https://explorer.bitcoin.com/tbch/tx/ae11339e0b435118a01b9c08c26a2e736cb7c9c78ab4c31e05a493aef74f776c

None of them has shown up in my wallet on my phone even though they have confirmed on the blockchain.

**Comment by micahriggan** [comment]
Both testnets are currently disabled, BCH tesnet was getting spammed with 32 mb blocks and BTC was getting spammed and reorg'd every few minutes. 

Ill see if we can get that turned back on soon

**Comment by offset25** [comment]
yes running into the same issue. Also tried with bitpay and no luck. Trying to test integration with bitpay.

**Comment by prashantpawar** [comment]
This clearly can be handled a lot better in the UI, I wasted a lot of time waiting for tx to arrive and then trying it from multiple faucets. This was a very frustrating experience.

**Comment by guillaumeviel** [comment]
I also would like this to be handled better. A warning that it is disabled for the time being would be useful and prevent people from wasting time.
